Framework/Algorithms/test/ScaleXTest.h

```diff
@@ -110,20 +114,17 @@ class ScaleXTest : public CxxTest::TestSuite {
 
       for (size_t j = 0; j < xsize; ++j) {
         if (factor > 0) {
```

Comment:
@DimitarTasev you can speak to me or @quantumsteve  if the change suggested is not obvious to you.